docs/index.md

```diff
@@ -1,409 +1,591 @@
-
-
 ## Getting started
 
-1. Install a language server from the list below, ensuring it can be started from the command line (is in your PATH)
-
-2. Run "LSP: Enable Language Server" from Sublime's Command Palette to allow the server to start.
-
+1. Install a language server from the list below, ensuring it can be started from the command line (is in your PATH).
+2. Run "LSP: Enable Language Server Globally" or "LSP: Enable Lanuage Server in Project" from Sublime's Command Palette to allow the server to start.
 3. Open a document in your language - if the server starts its name will be in the left side of the status bar.
 
 
-### Javascript/Typescript<a name="jsts"></a>
+## About LSP
 
-Different servers wrapping microsoft's typescript services, most support plain javascript:
+The *Language Server Protocol* is a specification about the communication protocol for use between text editors or IDEs and *language servers* - tools which provide language-specific features like autocomplete, go to definition, or documentation on hover.
```

Comment:
```suggestion
The *Language Server Protocol* is a specification of a protocol that aims to standardize the way a *language server* and a client (here this package) communicate with each other in order to provide language-specific features like auto-completion, goto definition, documentation on hover ... in a specific development tool (here Sublime Text).
```

Reply:
Hm... I avoided the word "client" here to prevent confusion with the `"clients"` key in the settings, which is used for the list of servers. This sentence and the next one are basically just to explain for new users that the LSP package depends on separate language servers and doesn't provide these features by itself.